### PR TITLE
Redesign the Learn Landing Page

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Redesign the Learn landing page.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+permalink: /learn-beta2/
+intro: Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below.
+redirect_from:
+- /learn-beta2
+---
+## Getting Started
+
+### [Language Introduction Video](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s)
+
+Video presentation of above which people can use as a sample of how to present the one hour deck.
```
Comment (dakshitha) [inline documentation.md]:
```suggestion
A video introduction to the Ballerina Programming Language and what's new in Swan Lake.
```

Comment (praneesha) [reply documentation.md]: Fixed.

File: documentation.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+### [Language Taste Screencasts](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s)
+
+End-to-end video screencasts like the one Manu has done on writing a service.
```
Comment (dakshitha) [inline documentation.md]: End-to-end screencasts covering popular use cases that give a taste of the Ballerina language.

File: documentation.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+### [Language Walkthrough Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
+
+Slide deck explaining the language.
```
Comment (dakshitha) [inline documentation.md:23]:
```suggestion
A comprehensive reference slide deck explaining the language.
```

File: documentation.md (hunk 2 above)
Comment (praneesha) [reply documentation.md]: Fixed.

File: documentation.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+### [Language Walkthrough Video](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s) 
+
+Video recording of the language slide deck.
```
Comment (dakshitha) [inline documentation.md:27]:
```suggestion
A video recording of the language slide deck.
```

File: documentation.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+### [Learn by Example](/learn/by-example/)
+
+Guided path to learn the language by a series of examples.
```
Comment (dakshitha) [inline documentation.md:31]:
```suggestion
A guided path to learn the language through a series of examples.
```

File: documentation.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+## Concepts
+
+### [Why Ballerina](/learn/why-ballerina/)
+
+Explains the unique aspects of Ballerina.
+
+### [User Guide](/learn/user-guide/)
+
+Series of guides on various things you need to do during the Ballerina development cycle.
```
Comment (dakshitha) [inline documentation.md:41]:
```suggestion
A series of guides on various things you need to do during the Ballerina development cycle.
```

File: documentation.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+## References
+
+### [Language Introduction Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
+
+One hour presentation on the language that 3rd parties can use to talk about the language.
```
Comment (dakshitha) [inline documentation.md]:
```suggestion
Presentation slides on the Ballerina language that anyone can use to talk about the language.
```

File: documentation.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+### [Language Guide](/learn/language-concepts/)
+
+Prose extracted from slide deck and videos, with expanded examples.
```
Comment (dakshitha) [inline documentation.md]:
```suggestion
An elaborate textual guide to the Ballerina language. 
```

File: documentation.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+### [Library Documentation](/learn/api-docs/)
+
+All library API docs.
+
+### [Reference Guide by Examples](/learn/by-example/)
+
+Series of examples that serve as a reference guide for each language construct & concept.
```
Comment (dakshitha) [inline documentation.md]:
```suggestion
A series of examples that serve as a reference guide for each language construct & concept.
```

**Review by dakshitha — CHANGES_REQUESTED** [review]
Please modify as suggested.

File: documentation.md (hunk 7 above)
Comment (praneesha) [reply documentation.md]: Fixed.

File: documentation.md (hunk 9 above)
Comment (praneesha) [reply documentation.md]: Fixed.

**Review by dakshitha — APPROVED** [review]

**Review by dakshitha — APPROVED** [review]

**Review by dakshitha — CHANGES_REQUESTED** [review]
Remove & and use and

**Review by dakshitha — APPROVED** [review]

**Pull request merged**